WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Creation/ProductRowViewModelTests.swift

```diff
@@ -343,6 +343,73 @@ final class ProductRowViewModelTests: XCTestCase {
         XCTAssertEqual(viewModel.quantity, 6)
     }
 
+    func quantity_decrementer_disabled_at_minimum_quantity_when_removeProductIntent_is_nil() {
```

Comment:
I think test cases need the `test` prefix in the function name in order to be run (similar for other test cases in this class)
```suggestion
    func test_quantity_decrementer_disabled_at_minimum_quantity_when_removeProductIntent_is_nil() {
```

Reply:
Example: the test case doesn't have the check box in Xcode as in the previous test case

<img width="1066" alt="Screenshot 2023-11-20 at 4 46 45 PM" src="https://github.com/woocommerce/woocommerce-ios/assets/1945542/3b2e6b42-fe96-44cb-8482-14dc315c93fa">

Reply:
[Reference](https://developer.apple.com/library/archive/documentation/DeveloperTools/Conceptual/testing_with_xcode/chapters/04-writing_tests.html):

> A test method is an instance method of a test class that begins with the prefix test, takes no parameters, and returns void